stripe/src/main/java/com/stripe/android/paymentsheet/BottomSheetController.kt

```diff
@@ -5,60 +5,52 @@ import android.view.ViewGroup
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.distinctUntilChanged
 import com.google.android.material.bottomsheet.BottomSheetBehavior
-import kotlinx.coroutines.CoroutineScope
-import kotlinx.coroutines.delay
-import kotlinx.coroutines.launch
 
 internal class BottomSheetController(
-    private val bottomSheetBehavior: BottomSheetBehavior<ViewGroup>,
-    private val lifecycleScope: CoroutineScope
+    private val bottomSheetBehavior: BottomSheetBehavior<ViewGroup>
```

Comment:
Are there any unit tests that should be added for these changes?

Reply:
The tests cover the sheet state, the layout animations being enabled and isFitToContents = false once it's expanded. I think that covers the main attributes that could be changed by accident.